README.md

```diff
@@ -115,11 +123,32 @@ The ‘role’ key maps to the ‘module:rRoleName’, which in this case is ‘
 
 The remaining key: value pairs map to the fields of the role as described in the STL file.
 
-### 3.4. test
+### 3.4. constants
+The ‘constants’ field must be a dictionary. Each entry in the dictionary defines a declared constant described in the stl_files listed in the ‘stl_files’ field. In the above example, there [example_base.stl](example/example_base.stl) contains the following:
+```
+example_base.stl:
+
+module example;
+...
+const string kHelloWorld;
```

Comment:
What is the initial value of this case? Or, does parser checks whether the constant is initialized somewhere?